com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -448,6 +463,8 @@ public void LazyInitialize()
                 new int[m_ActuatorManager.NumDiscreteActions]
             );
 
+            m_Info.teamManagerId = m_TeamManager == null ? -1 : m_TeamManager.GetId();
```

Comment:
So the convention is that -1 is "no team".

Reply:
It actually should be 0 since it's what it would be in proto if using old version of C#.